src/sasctl/pzmm/write_json_files.py

```diff
@@ -273,20 +271,18 @@ def writeFileMetadataJSON(cls, modelPrefix, jPath=Path.cwd(), isH2OModel=False):
         Writes a file metadata JSON file pointing to all relevant files.
 
         Parameters
-        ---------------
+        ----------
         modelPrefix : string
-            The variable for the model name that is used when naming model files.
-            (for example, hmeqClassTree + [Score.py || .pickle]).
+            The variable for the model name that is used when naming model files. (For example, hmeqClassTree +
+            [Score.py | .pickle]).
         jPath : string, optional
-            Location for the output JSON file. The default value is the current
-            working directory.
+            Location for the output JSON file. The default value is the current working directory.
         isH2OModel : boolean, optional
-            Sets whether the model metadata is associated with an H2O.ai model.
-            If set as True, the MOJO model file will be set as a score resource.
-            The default value is False.
+            Sets whether the model metadata is associated with an H2O.ai model. If set as True, the MOJO model file
+            will be set as a score resource. The default value is False.
 
         Yields
-        ---------------
+        ------
```

Comment:
Same issue as with writeVarJSON() docstring, the Yields section is inappropriate here.

Reply:
Resolved.